### ISSUE TITLE
Change Menu name "Privacy" to "GDPR Framework"

### ISSUE THREAD

**Issue by teolaz** [codelight-eu/gdpr-framework#7]
Hi,
i think with the new WP update, it's better to avoid the situation below (my WP is in Italian):

![image](https://user-images.githubusercontent.com/19909323/40364830-8793919c-5dd3-11e8-9f02-59c124904736.png)
![image](https://user-images.githubusercontent.com/19909323/40364848-9252a064-5dd3-11e8-9eb9-149de03d9881.png)

**Comment by indrek-k** [comment]
Yes, that's a great point. We'll change it or possibly merge with WP's own Privacy page.
